packages/e2e/test/k6/scenarios/web-socket.test.js

```diff
@@ -1,15 +1,25 @@
+// cSpell:ignore loadimpact
+
 import * as k6Utils from '../../../../util-dev/dist/cjs/k6-utils.js';
 import { Counter, Trend } from 'k6/metrics';
+import { SharedArray } from 'k6/data';
 import { check } from 'k6';
 import ws from 'k6/ws';
 
-// eslint-disable-next-line no-undef
-const { WALLETS } = Object.assign({ WALLETS: '10000' }, __ENV);
-// eslint-disable-next-line no-undef
-const dut = k6Utils.getDut(__ENV);
+const { HD_ACTIVE_ADDR_COUNT, HD_MAX_TX_HISTORY, TARGET_NET, WALLETS } = Object.assign(
+  { HD_ACTIVE_ADDR_COUNT: '10', HD_MAX_TX_HISTORY: '100', TARGET_NET: 'mainnet', WALLETS: '100' },
+  // eslint-disable-next-line no-undef
+  __ENV
+);
```

Comment:
```suggestion
const { HD_ACTIVE_ADDR_COUNT, HD_MAX_TX_HISTORY, TARGET_NET, WALLETS } = 
  { HD_ACTIVE_ADDR_COUNT: '10', HD_MAX_TX_HISTORY: '100', TARGET_NET: 'mainnet', WALLETS: '100', ...__ENV};
```

Reply:
K6 JavaScript environment doesn't support the proposed syntax. 😞